### PR TITLE
Discard grep output

### PR DESCRIPTION
**Pull request by kirelagin**
Without this grep prints successful matches on the terminal which makes the output of `git annex` looks weird.

### CODE REVIEW

File: git-annex-remote-rclone [hunk 1]
```diff
@@ -199,14 +199,14 @@ while read -r line; do
 			# Some rclone backends support multiple
 			# files under one name.
 			if check_result=$(rclone size "$LOC$key" 2>&1) &&
-			    echo $check_result|grep -E 'Total objects: [1-9][0-9]* Total size' >&2 &&
-			    ! echo $check_result|grep 'Total size: 0 (0 bytes)' >&2; then
+			    echo $check_result|grep -E 'Total objects: [1-9][0-9]* Total size' >/dev/null &&
+			    ! echo $check_result|grep 'Total size: 0 (0 bytes)' >/dev/null; then
```
Comment (yarikoptic) [inline git-annex-remote-rclone:203]: and why not to use `grep -q` to start with?
works even on mac
```
yoh@datalads-imac2 ~ % echo 123 | grep -q 1 ; echo $?
0
yoh@datalads-imac2 ~ % echo 123 | grep  1 ; echo $? 
123
0
yoh@datalads-imac2 ~ % uname -a
Darwin datalads-imac2.local 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64
yoh@datalads-imac2 ~ % which grep
/usr/bin/grep
```

Comment (yarikoptic) [inline git-annex-remote-rclone:203]: and why not to use `grep -q` to start with?
works even on mac
```
yoh@datalads-imac2 ~ % echo 123 | grep -q 1 ; echo $?
0
yoh@datalads-imac2 ~ % echo 123 | grep  1 ; echo $? 
123
0
yoh@datalads-imac2 ~ % uname -a
Darwin datalads-imac2.local 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64
yoh@datalads-imac2 ~ % which grep
/usr/bin/grep
```

Comment (yarikoptic) [reply git-annex-remote-rclone:203]: FWIW: and seems to work on windows (in "git shell"), also tested the /dev/null:
```
DataLad@DESKTOP-1FTT289 MINGW64 ~
$ echo 123 | grep -q 1 ; echo $?
0
(base)
DataLad@DESKTOP-1FTT289 MINGW64 ~
$ echo 123 | grep  1 ; echo $?
123
0
(base)
DataLad@DESKTOP-1FTT289 MINGW64 ~
$ echo 123 | grep  1 > /dev/null; echo $?
0
(base)
DataLad@DESKTOP-1FTT289 MINGW64 ~
$ echo 123 | grep  -q 1 > /dev/null; echo $?
0
```

**Comment by yarikoptic** [comment]
FWIW, looks kosher to me one way (as is) or another (with `-q`)  for the effect of the command.  Might have helped with #44

**Comment by DanielDent** [comment]
Thank you all for your contributions. This led to #49 which has now been merged.
